plugins/core/viame_train_detector.cxx

```diff
@@ -43,12 +43,18 @@
 #include <vital/algo/algorithm_factory.h>
 #include <vital/algo/train_detector.h>
 #include <vital/algo/detected_object_set_input.h>
+#include <vital/config/config_block_io.h>
+#include <arrows/ocv/image_container.h>
+#include <arrows/ocv/image_io.h>
 
 #include <boost/filesystem.hpp>
 #include <boost/shared_ptr.hpp>
 #include <boost/algorithm/string.hpp>
 #include <boost/lexical_cast.hpp>
 
+#include <sprokit/pipeline/process_exception.h>
+#include <sprokit/processes/adapters/embedded_pipeline.h>
+#include <sprokit/processes/adapters/adapter_types.h>
```

Comment:
Minor style: try to keep all includes grouped together with the same things (just add a space before the C++ includes)

Reply:
Fixed and rebased.